### ISSUE TITLE
Pylance doesn't work anymore (when a ".py" file is present)

### ISSUE THREAD

**Issue by TFCx** [microsoft/pylance-release#810]
## Environment data

-   Language Server version: 1.12.2
-   OS and version:
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal

-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.5 64bits

## Expected behaviour

Pylance should analyse semantically the code. It works with 12.1.

![image](https://user-images.githubusercontent.com/9135500/103773882-06ee5d80-502c-11eb-8ca6-4bfdc3a5ca9a.png)

## Actual behaviour

Semantic analysis is not working and even token inspection doesn't return anything.

![image](https://user-images.githubusercontent.com/9135500/103773544-84fe3480-502b-11eb-8a6a-1a7e4ac877d9.png)

## Logs

```
[Error - 2:25:28 PM] stubPath /home/jean-david/Work/Anatoscope/src/circle/articulator/typings is not a valid directory.
[Info  - 2:25:28 PM] Assuming Python version 3.8
[Info  - 2:25:28 PM] Assuming Python platform Linux
[Info  - 2:25:28 PM] Searching for source files
[Info  - 2:25:28 PM] Background analysis(1) root directory: /home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist
[Info  - 2:25:28 PM] Background analysis(1) started
Background analysis message: setConfigOptions
Background analysis message: setTrackedFiles
Error: Debug Failure. /home/jean-david/Work/Anatoscope/src/circle/articulator/legacy/Calibration/.py is not a source file
    at P._addToSourceFileListAndMap (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/analyzer/program.ts:2075:13)
    at P.addTrackedFile (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/analyzer/program.ts:245:14)
    at forEach (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/analyzer/program.ts:211:18)
    at Array.forEach (<anonymous>)
    at P.addTrackedFiles (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/analyzer/program.ts:210:19)
    at P.setTrackedFiles (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/analyzer/program.ts:195:14)
    at B.onMessage (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/backgroundAnalysisBase.ts:354:50)
    at B.onMessage (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/server.bundle.js:1:32759)
    at MessagePort.<anonymous> (/home/jean-david/.vscode/extensions/ms-python.vscode-pylance-2020.12.2/dist/pyright-internal/src/backgroundAnalysisBase.ts:273:66)
    at MessagePort.emit (events.js:223:5)

```

## Code Snippet / Additional information

Semantic analysis seems to crash when encountering a file name ".py". If this file is renammed "x.py", 12.2 seems to work as intented.

**Comment by erictraut** [comment]
This will be fixed in the next release of pylance.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.1.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202110-6-january-2021

**Comment by Nicialy** [comment]
Hello, why Pylance dont work in Kali Linux?
